consensus/polybft/bridge_event_manager.go

```diff
@@ -317,7 +329,7 @@ func (b *bridgeEventManager) getAggSignatureForBridgeBatchMessage(blockNumber ui
 	votes, err := b.state.BridgeMessageStore.getMessageVotes(
 		pendingBridgeBatch.Epoch,
 		bridgeBatchHash.Bytes(),
-		b.sourceChainID)
+		b.externalChainID)
```

Comment:
Since this is called from `buildBridgeBatch` which we use for building both blade originated and external originated batches, you will need to pass the chain id to the `getAggSignatureForBridgeBatchMessage` in order to get message votes.